src/app/(drawerPages)/badges/[id]/page.tsx

```diff
@@ -45,61 +60,143 @@ export default function BadgeDetails({ params }: Props) {
     }
   }, [isDisconnected, router]);
 
+  useEffect(() => {
+    drawerStates.badgeActions.bottom = true;
+  }, [currentBadge, drawerStates.badgeActions]);
+
+  const ctaButton = useMemo(() => {
+    if (isOwned) return null;
+    if (!currentBadge) return null;
+    if (currentBadge.type === BadgeTypeEnum.IRL) {
+      return null;
+    } else if (currentBadge.type === BadgeTypeEnum.Online) {
+      return (
+        <Button variant="contained" href={currentBadge.ctaUrl}>
+          Mint
+        </Button>
+      );
+    }
+  }, [currentBadge, isOwned]);
+
+  const content = useMemo(() => {
+    if (currentBadge) {
+      const title = currentBadge?.name;
+      const [lat, lng] = currentBadge.latLng
+        .split(',')
+        .map((coord) => parseFloat(coord));
+
+      return (
+        <Stack
+          gap={1}
+          px="20px"
+          py="24px"
+          sx={{
+            background: 'white',
+          }}
+          marginBottom={['5rem', 'unset']}
+        >
+          {/* <ToolBar title={title} onDismiss={() => setIsMapOpen(false)} /> */}
+          <Stack direction="column" gap={1}>
+            <Stack direction="column" gap={1}>
+              <Stack flexDirection="row" justifyContent="space-between">
+                <Text color="black" fontWeight={400} fontSize="24px">
+                  {title}
+                </Text>
+                {isMobile ? (
+                  <Link href={getNavigationUrl(currentBadge?.latLng)}>
+                    <Pill backgroundColor="black">
+                      <Text color="white">Get directions</Text>
+                    </Pill>
+                  </Link>
+                ) : (
+                  <Pill backgroundColor="black">
+                    <Text color="white">Get directions</Text>
+                  </Pill>
+                )}
```

Comment:
shouldn't this check for if badge is IRL or not?

Reply:
If it is IRL, we show 'get directions' and if not, we show 'mint now' is this correct?